internal/v2/fake_s2av2_service/fake_s2av2_service.go

```diff
@@ -0,0 +1,31 @@
+package main
+
+import (
+	"net"
+	"flag"
+	"log"
+	"google.golang.org/grpc"
+	"github.com/google/s2a-go/internal/v2/fake_s2av2"
+	s2av2pb "github.com/google/s2a-go/internal/proto/v2/s2a_go_proto"
+)
+
+var (
+	listenPort = flag.String("listen_port", ":8080", "Fake S2Av2 service address port.")
```

Comment:
nit: Can we just call the variable and flag "port" instead of "listenPort"? The former is more common.